src/win32/i_steam.cpp

```diff
@@ -355,7 +361,6 @@ TArray<FString> I_GetBethesdaPath()
 	TArray<FString> result;
 	static const char* const bethesda_dirs[] =
 	{
-		// TODO Does the 2024 Update affect Bethesda Launcher?
```

Comment:
To answer @vitalkanev 's question: Bethesda.net Launcher closed down in 2022, so no.